app/controllers/app.articles.controller.js

```diff
@@ -1,15 +1,19 @@
-const { fetchArticles, fetchArticle } = require("../models/app.articles.model");
+const {
+  fetchArticles,
+  fetchArticleComments,
+} = require("../models/app.articles.model");
 
 exports.getArticles = (req, res, next) => {
   fetchArticles().then((articles) => {
     res.status(200).send({ articles });
   });
 };
 
-exports.getArticle = (req, res, next) => {
-  fetchArticle(req.params)
-    .then((article) => {
-      res.status(200).send({ article: article });
+exports.getArticleComments = (req, res, next) => {
+  const { article_id } = req.params;
+  fetchArticleComments(article_id)
+    .then((comments) => {
+      res.status(200).send({ [`article_id${article_id} comments`]: comments });
```

Comment:
The name of this key does not need to be something like this, simply having comment as the name of this key is enough for this.